### PR TITLE
Example: Response schema of get account.

### PR DESCRIPTION
**Pull request by maksymmateiuk**
**⚠️Please do not make any changes to spec files in this repository!⚠️**

**All spec changes should be done in the [api-specs](https://github.com/Bandwidth/api-specs) repository.**

**Please follow the [Guide](https://bandwidth-jira.atlassian.net/wiki/spaces/DX/pages/4098359409/How+to+Update+API+Specifications+and+Contribute+to+Developer+Documentation) for contributing to our developer documentation.**

### CODE REVIEW

**Review by mrmeyers99 — DISMISSED** [review]
LGTM

**Review by ajrice6713 — APPROVED** [review]

File: site/specs-temp/numbers.json [hunk 1]
```diff
@@ -17816,7 +17819,260 @@
...
+          "ResetAddressFields" : {
+            "type" : "boolean"
+          },
+          "Latitude" : {
+            "type" : "string"
+          },
+          "Longitude" : {
+            "type" : "string"
+          },
+          "AddressType" : {
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:17901]: Can we declare this enum as its own schema?

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

File: site/specs-temp/numbers.json (hunk 1 above)
Comment (ajrice6713) [reply site/specs-temp/numbers.json:17901]: See the `Reusable Enums` section [here](https://swagger.io/docs/specification/data-models/enums/) 

reason being is that the client generators function a little more nicely with the resuable enums

File: site/specs-temp/numbers.json [hunk 2]
```diff
@@ -17816,7 +17819,260 @@
...
+          "PlusFour" : {
+            "type" : "string"
+          },
+          "County" : {
+            "type" : "string"
+          },
+          "Country" : {
+            "type" : "string"
+          },
+          "LegacyAddressId" : {
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:17884]: Do we want this legacy parameter defined? 

we probably need a description explaining what this is

File: site/specs-temp/numbers.json [hunk 3]
```diff
@@ -17816,7 +17819,260 @@
...
+        "properties": {
+          "Account": {
+            "$ref": "#/components/schemas/Account"
+          },
+          "AddressErrorDescription": {
+            "$ref": "#/components/schemas/AddressErrorDescription"
+          }
+        }
+      },
+      "Address" : {
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:17834]: We might want descriptions and examples here as well as nullable and default values(if any) defined

File: site/specs-temp/numbers.json (hunk 1 above)
Comment (maksymmateiuk) [reply site/specs-temp/numbers.json:17901]: Good point!

File: site/specs-temp/numbers.json (hunk 2 above)
Comment (maksymmateiuk) [reply site/specs-temp/numbers.json:17884]: I can remove this field from the schema, I don't know do we need it.

File: site/specs-temp/numbers.json (hunk 3 above)
Comment (maksymmateiuk) [reply site/specs-temp/numbers.json:17834]: Examplex, nullable default values for response schema?

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Pull request merged**